.github/workflows/github-actions-release.yml

```diff
@@ -47,28 +93,39 @@ jobs:
         #
         # The second `git checkout HEAD^` is to prevent errors from the post jobs like "Prepare node".
         run: |
+          SOURCE_SHA=$(git rev-parse HEAD);
+          TAG_NAME="${{ github.event.release.tag_name }}"
+          if [ "$TAG_NAME" = '' ]; then
+            TAG_NAME="${{ fromJSON(needs.Setup.outputs.release).tag_name }}"
```

Comment:
:grey_question: :nail_care: Maybe we could have a step that would resolve that condition once, and we would just reuse its output for checkout, and here?

Reply:
Thanks for the great idea! I adjusted this in https://github.com/woocommerce/grow/pull/22/commits/171be1d740de386a7ba9fefbb27872796c239600.